site/en/guide/migrate/saved_model.ipynb

```diff
@@ -303,7 +303,7 @@
       "source": [
         "### Save and export a SavedModel defined with tf.Module\n",
         "\n",
-        "To export your model in TensorFlow 2, you must define a `tf.Module` or a `tf.keras.Model` to hold all of your model's variables and functions. Then, you can call `tf.saved_model.save` to create a SavedModel. Refer to [Saving a custom model](../../guide/saved_model#saving_a_custom_model) in the [Using the SavedModel format](../../guide/saved_model) guide to learn more."
```

Comment:
Can you please leave the anchors in-place on the links?

e.g. `[Using the SavedModel format](../saved_model.ipynb#saving_a_custom_model)`

They are useful when the docs are hosted on tensorflow.org, and cause no harm when viewed in places that don't support them.

Reply:
As discussed with @markmcd , unless it's custom, it's better not to.